webvtt.html

```diff
@@ -1171,7 +1168,7 @@
     contain the substring "<code>--></code>" (U+002D HYPHEN-MINUS, U+002D HYPHEN-MINUS, U+003E
     GREATER-THAN SIGN).</p>
 
-    <p>A <dfn>WebVTT cue</dfn> consists of the following components, in the given order:</p>
```

Comment:
Looking at the WHATWG HTML spec, Ian also always links back to the "text track cue" even when in the parsing sections of the video element. He doesn't distinguish between the concept and its instantiation. Should we really be separating the two?